Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -517,8 +517,7 @@ public override void Disconnect()
         /// <returns></returns>
         private void SetPendingOrder(Order order)
         {
-            // only save off clones!
-            _pending[order.Id] = order.Clone();
```

Comment:
Can we please add an order immutability regression test? I don't think we explicitly cover that -- something like we try 10 different ways to edit the order value but from QCAlgorithm layer it should be impossible

Reply:
Added a regression algorithm for that, from what I see, it seems like I covered the only ways to try and modify the order. Since you only get the ticket in the algorithm I had to go through `Transactions`!